otelcolbuilder/Makefile

```diff
@@ -102,7 +102,7 @@ endif
 
 .PHONY: _builder
 _builder:
-	$(eval VERSION ?= $(shell git describe --tags --abbrev=5 --match "v[0-9]*"))
+	$(eval VERSION ?= $(shell git describe --tags --abbrev=5 --match "v[0-9]*"| sed 's/^v//'))
```

Comment:
Remove the prefix (v) from the version

Reply:
cc: @amdprophet

Reply:
This it out of date: https://github.com/SumoLogic/sumologic-otel-collector/blob/main/otelcolbuilder/Makefile. You may want to merge main into this branch before merging as there are quite a few changes to the Makefile.

Reply:
I rebased and pushed the changes.